### PR TITLE
Release for reviewer feedback.

### PR DESCRIPTION
**Pull request by wpschaub**

### CODE REVIEW

File: content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md
Comment (AndreKaminski) [inline content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md:1]: Looks good. In the 'avoid' list, I would include also - rigid, monolithic architecture preventing evolution of the system

File: content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+Gone are the days of simply tossing tasks over the wall to different roles like content editors, developers, testers, admins, and so forth. The only boundary we acknowledge are the faintly dotted lines that delineate teams and groups of teams within our organization. Beyond that, we operate and own QUALITY as one unified entity.
+
+# Think product team!
+
+When you operate with a healthy [DevOps](https://www.donovanbrown.com/post/what-is-devopss) mindset, you will also be familiar with the "product mindset", which implies that we, as a team, are accountable for our solution from ideation to deprecation. Until we switch off the solution lights, we own the accountability for QUALITY!
+
```
Comment (MerlinDarkstar) [inline content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md:44]: Pride, feeling like you've created and contributed to something you are proud of - that is the inner spirit we need to kindle.

File: content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+- **Confidence** created when everyone follows set of guidelines to follow, enciuraging creative problem-solving and innovation.
+- **Cost-Efficiency** by avoiding previous mistakes, saving time, and money.
+- **Forward-thinking** promoted by promoting practices that are known, proven, and sustainable. This benefits the maintenance and evolution of solutions over time.
+- **Avoid rigid and monolithic architectures** which impede the system's adaptability and evolution, especially in today's rapidly changing technological landscape. 
+
+There are many more examples, but I need to touch on my 80:20 rule and then stop venting.
+
+# 80:20 planning rule
+
+Lastly, it is important to discuss how we plan and allocate our time. If we aim to fill every minute of our schedule, we are setting ourselves up to work beyond our capacity. This leaves us scrambling to address unexpected issues, bugs, and the lingering technical debt that tends to resurface repeatedly. I know it, you know it - it is time to smarter and more efficiently. We must move away from burnout and start embracing a healthier work:life balance.
```
Comment (MerlinDarkstar) [inline content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md:76]: time to *work* smarter and more efficiently - optimize and capitalize on down time.

**Review by MerlinDarkstar — DISMISSED** [review]

File: content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md (hunk 2 above)
Comment (wpschaub) [reply content/embedding-a-quality-driven-engineering-mindset-in-our-dna.md:76]: Done

**Review by danielrbroderick — APPROVED** [review]

**Review by MerlinDarkstar — APPROVED** [review]
Love it!

**Review by AndreKaminski — APPROVED** [review]
spelling mistake in "Wotk smarter and more efficiently"

**Comment by wpschaub** [comment]
Thank you @AndreKaminski. Will fix typo in the publication PR.

**Pull request merged**